data/store/mongo/mongo.go

```diff
@@ -1081,8 +1087,8 @@ func (d *DataRepository) GetLastUpdated(ctx context.Context, id string) (time.Ti
 	}
 
 	if dataSet != nil {
-		lastUpload, err = time.Parse(time.RFC3339Nano, *dataSet[0].CreatedTime)
-		lastData, err = time.Parse(time.RFC3339Nano, *dataSet[0].Time)
+		lastUpload = *dataSet[0].CreatedTime
+		lastData = *dataSet[0].Time
 	} else {
 		return lastUpload, lastData, ErrorUserCbgNotFound
```

Comment:
This is a code smell - consider returning a struct instead.

Reply:
this is part of the summary code, and will be addressed there.